lib/protip/wrapper.rb

```diff
@@ -201,6 +208,8 @@ def to_protobuf_value(field, value)
           value
         elsif converter.convertible?(field.subtype.msgclass)
           converter.to_message value, field.subtype.msgclass
+        elsif nested_resources.has_key?(field.name.to_sym)
+          value.message
```

Comment:
this assumes that a resource is passed in as `value`, but currently, lots of our associations pass in messages

i guess it will be fine, since those "associations" won't be declared as associations yet. i guess when we declare an association, we need to carefully change that model's `as_proto` ? (i'm not sure we have very good integration test coverage of protip stuff, so we'll have to be very careful)